corporate/lib/stripe.py

```diff
@@ -575,6 +583,10 @@ def billing_session_url(self) -> str:
     def get_customer(self) -> Optional[Customer]:
         return get_customer_by_realm(self.realm)
 
+    @override
+    def current_count_for_billed_licenses(self) -> int:
+        return get_latest_seat_count(self.realm)
```

Comment:
This is used a bunch in the code base, so instead of moving `get_seat_count` or `get_latest_seat_count` to be part of the `BillingSession` class, I thought it made more sense to just alias it here for the realm billing session / customer case.

Reply:
Yeah, that makes sense, especially since the alternative methods will do something completely different here.